### PR TITLE
Enhancements/customizable expirations

### PR DESCRIPTION
**Pull request by jakewrfoster**

### CODE REVIEW

File: client/js/global.js [hunk 1]
```diff
@@ -48,7 +49,7 @@ function fmOverlay() {
     const date = new Date();
 
     // set cookie for 2 hours
-    date.setTime(date.getTime() + (2 * 60 * 60 * 1000));
+    date.setTime(date.getTime() + (expiration * 60 * 60 * 1000));
```
Comment (mslinnea) [inline client/js/global.js:52]: any chance of expiration not being a number here?

**Review by mslinnea — APPROVED** [review]
Looks good, just one question. 🐴

File: client/js/global.js (hunk 1 above)
Comment (jakewrfoster) [reply client/js/global.js:52]: this value goes through `absint()` coming into the meta

Comment (jakewrfoster) [reply client/js/global.js:52]: I will absint on output to be sure as well. doesn't hurt :) thanks!

**Pull request merged**